hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -55,3 +57,11 @@ objects:
     account-limit: "${ACCOUNT_LIMIT}"
     root: ${ROOT_OU_ID}
     base: ${BASE_OU_ID}
+
+- apiVersion: v1
+  kind: AccountPool
+  metadata:
+    name: aws-account-operator-accountpool
+    namespace: aws-account-operator-accountpool-namespace
```

Comment:
I think this can be changed to `aws-account-operator` or be removed

Reply:
oops that was a mistake on my part, nice catch